### PR TITLE
refactor 1330: use new secrets model

### LINKED ISSUE #1330: Use our new parsers.secrets model for existing secrets
@elazarg FYI - opened a different issue for this one

### PR DESCRIPTION
**Pull request by rsperer**
Used new secrets model wherever possible, also moved secrets.py to anyway directory. Closes #1330

### CODE REVIEW

File: anyway/utilities.py [hunk 1]
```diff
@@ -36,16 +36,16 @@ def init_flask():
     )
     app.config.from_object(config)
     app.config["BABEL_TRANSLATION_DIRECTORIES"] = os.path.join(_PROJECT_ROOT, "translations")
-    if os.environ.get("PROXYFIX_X_FOR"):
+    if secrets.get("PROXYFIX_X_FOR"):
```
Comment (elazarg) [inline anyway/utilities.py:39]: Is it really a secret?

**Comment by elazarg** [comment]
Good work!

**Comment by elazarg** [comment]
The last commit (reformat with black) introduces changes in files that has nothing to do with the PR. Since this PR is sensitive, I think it's better to revert those files - files that are otherwise untouched - unless that would be too much trouble.

File: anyway/utilities.py (hunk 1 above)
Comment (rsperer) [reply anyway/utilities.py:39]: I don't know, not clear what's considered a secret and what isn't. 
Asking @atalyaalon

File: anyway/config.py [hunk 2]
```diff
@@ -10,7 +11,7 @@
 
 SQLALCHEMY_DATABASE_URI = os.getenv("DATABASE_URL", "postgresql://postgres@localhost/anyway")
 SQLALCHEMY_TRACK_MODIFICATIONS = True
-ENTRIES_PER_PAGE = os.environ.get("ENTRIES_PER_PAGE", 1000)
+ENTRIES_PER_PAGE = int(secrets.get_with_default("ENTRIES_PER_PAGE", "1000"))
```
Comment (elazarg) [inline anyway/config.py:14]: This also does not seem to be a secret - and `get_with_default` is a hint.

File: anyway/secrets.py [hunk 3]
```diff
@@ -12,7 +12,17 @@ def get(secret_name: str) -> str:
...
+
+
+def get_with_default(secret_name: str, value: str) -> str:
+    logging.info("Reading secret for " + repr(secret_name))
+    secrets_dir = "/run/secrets/"
+    if os.path.isdir(secrets_dir):
+        with open(secrets_dir + secret_name) as secret_file:
+            return secret_file.read()
+    logging.info("No secret dir found, returning default: " + value)
+    return value
```
Comment (elazarg) [inline anyway/secrets.py:25]: I don't think we need this function - when we have a secret, there shouldn't be any useful default.

Also, if we need the function, then `get` should use it, and assert on the result.

File: anyway/utilities.py [hunk 4]
```diff
@@ -36,16 +36,16 @@ def init_flask():
...
             app.wsgi_app,
             x_for=int(os.environ["PROXYFIX_X_FOR"]),
-            x_host=int(os.environ.get("PROXYFIX_X_HOST", "0")),
-            x_port=int(os.environ.get("PROXYFIX_X_PORT", "0")),
-            x_prefix=int(os.environ.get("PROXYFIX_X_PREFIX", "0")),
-            x_proto=int(os.environ.get("PROXYFIX_X_PROTO", "0")),
+            x_host=int(secrets.get_with_default("PROXYFIX_X_HOST", "0")),
+            x_port=int(secrets.get_with_default("PROXYFIX_X_PORT", "0")),
+            x_prefix=int(secrets.get_with_default("PROXYFIX_X_PREFIX", "0")),
+            x_proto=int(secrets.get_with_default("PROXYFIX_X_PROTO", "0")),
```
Comment (elazarg) [inline anyway/utilities.py:48]: Same comment about `get_with_default` as above - it probably isn't secret.

**Comment by rsperer** [comment]
Due to the amount of reverts required and for clarity purposes, I am closing this PR and will open a new one

**Pull request closed without merging**